rebasehelper/specfile.py

```diff
@@ -144,6 +145,12 @@ def _update_data(self):
         # HEADER of SPEC file
         self.hdr = self.spc.sourceHeader
 
+        try:
+            # try to download old sources from Fedora lookaside cache
+            LookasideCacheHelper.download('fedpkg', os.path.dirname(self.path), self.get_package_name())
+        except (LookasideCacheError, DownloadError):
```

Comment:
I think raising only one type of exception from LookasideCacheHelper would be simpler, since we are not handling them differently.